stac/src/main/scala/StacItem.scala

```diff
@@ -0,0 +1,64 @@
+package geotrellis.server.stac
+
+import geotrellis.server.stac.Implicits._
+
+import cats.implicits._
+import geotrellis.contrib.vlm.RasterSource
+import geotrellis.vector.{io => _, _}
+import io.circe._
+
+case class StacItem(
+    id: String,
+    _type: String = "Feature",
+    geometry: Geometry,
+    bbox: TwoDimBbox,
+    links: List[StacLink],
+    assets: Map[String, StacAsset],
+    collection: Option[String],
+    properties: JsonObject
```

Comment:
I left it `JsonObject` because of the confusion around representing "some of many" and out of a reluctance to allow _any_ json. `Map[String, Json]` might be better, but I'm not sure how. I think probably deferring specifying Json better until we do the shapeless record magic makes the most sense

Reply:
#128, for posterity